### DISCUSSION TITLE
Add optional option to show `Description` of a feed on main page for one-liner Rss-feeds.

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [spacecowboy/Feeder#212] [Ideas]
For example https://aur.archlinux.org/rss/modified has one-liner descriptions. Instead of opening the feed, it would make it easier to see description on main page intead of opening the feed.

**Comment by KAGEYAM4** [comment]
It's already there in settings.

**Comment by spacecowboy** [comment]
> For example https://aur.archlinux.org/rss/modified has one-liner descriptions. Instead of opening the feed, it would make it easier to see description on main page intead of opening the feed.

just increase allowed lines in your settings and stop clicking on them?

**Reply by KAGEYAM4** [reply]
> It's already there in settings.

I think my phrasing was wrong. First i though there wasn't any feature for this problem but when i looked into settings it was there.


But now i think, the option name is little bit wrong -> ` Show only title`, in fact it changes the behavior of description being shown or not. `Title` will be shown wether the option is on or off.

**Reply by spacecowboy** [reply]
Fair point
